### PR TITLE
update course settings to use new backend

### PR DESCRIPTION
**Pull request by jivey**
course-settings will pass once https://github.com/openstax/tutor-js/pull/3585 is merged.

### CODE REVIEW

File: tutor/specs/e2e/course-settings.e2e.ts [hunk 1]
```diff
@@ -1,29 +1,28 @@
-import { Factory, Mocker, visitPage, setTimeouts } from './helpers'
+import { visitPage, setTimeouts, loginAs } from './helpers'
 
 describe('Course Settings', () => {
-    let numStudents = 5
     beforeEach(async () => {
+	await loginAs({ page, userName: 'reviewteacher' })
```
Comment (nathanstitt) [inline tutor/specs/e2e/course-settings.e2e.ts:5]: alignment looks off, but maybe that's just how the diff is rendered?

File: tutor/specs/e2e/course-settings.e2e.ts [hunk 2]
```diff
@@ -1,29 +1,28 @@
...
+	    await page.click('testEl=next-btn')
+
+	    await page.fill('#number-students', '1')
+	    await page.click('testEl=next-btn')
+
+	    await page.waitForSelector('.course-page')
+	    await page.click('#actions-menu')
+	    await page.click('[data-item=courseSettings]')
+
+	    await page.click('testEl=c-o-u-r-s-e-d-e-t-a-i-l-s-tab')
```
Comment (nathanstitt) [inline tutor/specs/e2e/course-settings.e2e.ts:25]: ug, I thought that'd been fixed so it didn't dasherize the test-id :(

File: tutor/specs/e2e/course-settings.e2e.ts [hunk 3]
```diff
@@ -83,6 +81,7 @@ describe('Course Settings', () => {
             await page.click('testEl=delete-course-btn')
             await expect(page).toHaveSelector('testEl=disabled-delete-course-message-warning')
             await expect(page).toHaveText('testEl=delete-course-message', 'leave')
+	    await page.click('button.close')
```
Comment (nathanstitt) [inline tutor/specs/e2e/course-settings.e2e.ts:84]: also alignment?

**Review by nathanstitt — APPROVED** [review]

File: tutor/specs/e2e/course-settings.e2e.ts (hunk 3 above)
Comment (jivey) [reply tutor/specs/e2e/course-settings.e2e.ts:84]: Something with my Emacs config on Windows prevented me from seeing this :( I'll fix 'em

File: tutor/specs/e2e/course-settings.e2e.ts (hunk 2 above)
Comment (jivey) [reply tutor/specs/e2e/course-settings.e2e.ts]: Fixed, was being caused by the tab name being all uppercase

**Pull request merged**